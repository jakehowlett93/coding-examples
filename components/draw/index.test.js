import Draw from './index';

describe('draw', () => {
	const state = {
		textGuess: 'hello',
		room: {
			roundType: 'draw'
		}
	};
	let returnValue;
	beforeEach(() => {
		returnValue = Draw({ state });
	});
	it('should draw the draw component', () => {
		expect(returnValue).toMatchSnapshot();
	});
});
