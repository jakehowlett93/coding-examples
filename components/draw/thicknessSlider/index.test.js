import ThicknessSlider from './index';

describe('thickness slider', () => {
	const state = {
		room: { roundType: 'draw' }
	};

	const actions = {
		onThicknessInputChanged: jest.fn()
	};
	let returnValue;
	beforeEach(() => {
		returnValue = ThicknessSlider({ state, actions });
	});
	it('should write the right component ', () => {
		expect(returnValue).toMatchSnapshot();
	});
});
