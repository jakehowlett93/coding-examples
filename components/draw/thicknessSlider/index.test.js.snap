// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`thickness slider should write the right component  1`] = `
<div>
  <input
    id="thickness"
    max="100"
    min="0"
    onChange={[Function]}
    step="10"
    type="range"
  />
</div>
`;
